providers/json-binding/src/main/java/org/jboss/resteasy/plugins/providers/jsonb/JsonBindingProvider.java

```diff
@@ -37,14 +43,52 @@
 @Priority(Priorities.USER-100)
 public class JsonBindingProvider extends AbstractJsonBindingProvider
         implements MessageBodyReader<Object>, MessageBodyWriter<Object> {
-
+   private static final boolean JSONB_DISABLED = Boolean.getBoolean("resteasy.jsonb.disable");
    @Override
    public boolean isReadable(Class<?> type, Type genericType,
                              Annotation[] annotations, MediaType mediaType) {
-      if (isGenericJaxb(type, genericType))
+      if (JSONB_DISABLED)
+      {
+         return false;
+      }
+      ResteasyConfiguration context = ResteasyProviderFactory.getContextData(ResteasyConfiguration.class);
+      if (context != null && Boolean.parseBoolean(context.getParameter("resteasy.jsonb.disable")))
+      {
+         return false;
+      }
+      if (context != null && Boolean.parseBoolean(context.getParameter("resteasy.jsonp.enable")))
```

Comment:
`Boolean.parseBoolean(context.getParameter("resteasy.jsonp.enable")` is used twice, it could be put into property, similar to JSONB_DISABLED.

Reply:
`resteasy.jsonp.enable` is mentioned in docs just in relation to Jackson2JsonpInterceptor

Should be docs updated also for JsonBindingProvider section to mention it ?

Reply:
`resteasy.jsonb.disable`  vs.  `resteasy.jsonp.enable` can be confusing

similar property names, jsonb disable, but jsonp enable. This can be misleading for end users, I switched them when I was typing this comment

Reply:
resteasy.jsonp.enable is read from context if context is settings and It's different from JSONB_DISABLED reading from system property.
I failed to find some documentation about resteasy.jsonp.enable and we need to fix this.
About the confusing property name , is the full name like resteay.jsonbindng.disable better ?